### DISCUSSION TITLE
I remove my db accidently, is there any solution to get backup file of it?

### DISCUSSION THREAD

**Discussion by deli-ght** [vercel/community#4668] [Help] [answered]
### Summary

There was a problem while querying the vercel postgres. So I tryed to check what is going on my db and entered `prisma migrate -dev` and the whole data was disappeared.
After db was cleared, query error was solved but my data are gone. Is there any way to find them back?

### Steps to Reproduce

Enter `pnpm prisma migrate -dev` on terminal and cancel it when it ask a name.

**Comment by amyegan** [comment, marked as answer]
🙁 Sorry you ran into trouble with `prisma migrate`. [Automated backup support](https://vercel.com/docs/storage/vercel-postgres/faq#automated-backup-support) is not currently available for Postgres databases. If you already have a backup copy of the data, you could [import a Postgres database into your Vercel Postgres](https://neon.tech/docs/import/import-from-postgres). Our Storage partner, Neon, has also provided a guide about [using Prisma Migrate](https://neon.tech/docs/guides/prisma-migrate).

[vercel.com/docs/storage/vercel-postgres/faq](https://vercel.com/docs/storage/vercel-postgres/faq#)
